pkg/endpoint/endpoint.go

```diff
@@ -2007,3 +1978,23 @@ func (e *Endpoint) Delete(monitor monitorOwner, ipam ipReleaser, manager endpoin
 
 	return errs
 }
+
+// SetDefaultConfiguration sets the default configuration options for its
+// boolean configuration options and for policy enforcement based off of the
+// global policy enforcement configuration options.
+func (e *Endpoint) SetDefaultConfiguration(restore bool) {
```

Comment:
It would be good to mention the handling when restore is true since that isn't clear. It's more verbose, but we could just leave that check outside this functions. It only matters at one of the two callsites (and would be the same 3 lines there instead.

Reply:
Added the comment.